### PR TITLE
mroytman/checklist loading state

### PR DESCRIPTION
**Pull request by MichaelRoytman**

### CODE REVIEW

File: src/components/CourseChecklist/displayMessages.jsx [hunk 1]
```diff
@@ -131,6 +131,11 @@ const messages = defineMessages({
     defaultMessage: 'uncompleted',
     description: 'Label that describes an uncompleted task',
   },
+  loadingChecklistLabel: {
```
Comment (MichaelRoytman) [inline src/components/CourseChecklist/displayMessages.jsx]: I was thinking perhaps we'd want "loading course launch checklist" instead or something, but I think that because the checklist has the header above the loading icon, it would be okay to leave it this way.

File: src/components/CourseOutlineStatus/displayMessages.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  loadingChecklistLabel: {
+    id: 'loadingChecklistLabel',
+    defaultMessage: 'Loading',
```
Comment (MichaelRoytman) [inline src/components/CourseOutlineStatus/displayMessages.jsx]: In this case, I wonder if this should be a little more explicit. Maybe "loading checklist score" or something?

File: src/components/CourseChecklist/displayMessages.jsx (hunk 1 above)
Comment (fysheets) [reply src/components/CourseChecklist/displayMessages.jsx]: Hm I see what you're saying. I wonder if it would it make sense to have the Icon be hidden and have the header read out: "headerText loading" if it's loading / "loaded" once the data has been rendered? Or do you think that would be too wordy? 

Same goes for the course outline & the Checklists heading there.
